samcli/cli/command.py

```diff
@@ -104,6 +105,114 @@ def _set_commands(package_names):
 
         return commands
 
+    def format_options(self, ctx: Context, formatter: HelpFormatter) -> None:
+        self.format_commands(ctx, formatter)
+        opts = []
+        for param in self.get_params(ctx):
+            rv = param.get_help_record(ctx)
+            if rv is not None:
+                opts.append(rv)
+
+        if opts:
+            with formatter.section(click.style("Global Options", bold=True)):
+                formatter.write_dl(opts)
+
+    @staticmethod
+    def format_help_str_command(help_text, limit: int = 45) -> str:
+        # TODO(sriram-mv): do something with limit for indentation.
+
+        return help_text.strip()
+
+    def format_commands(self, ctx, formatter):
```

Comment:
Would these methods be part of the UX library?

Reply:
Potentially, there maybe some things that we need to refactor based on what we want to move. I wanted to get a POC out there first.